src/data/markdown/docs/02 javascript api/07 k6-experimental/07 grpc.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "grpc"
+excerpt: "Experimental GRPC module"
+---
+
+The `k6/experimental/grpc` module is an extension of the [`k6/net/grpc`](/javascript-api/k6-net-grpc/). It provides a [gRPC](https://grpc.io/) client for Remote Procedure Calls (RPC) over HTTP/2. 
+
+The key-difference between the two modules is new `Stream` class, which provides client and server streaming support. Our long-term goal is to make this module part of k6 core, and long-term to replace the [`k6/net/grpc`](/javascript-api/k6-net-grpc/) module.
+
+| Class/Method | Description |
+|--------------|-------------|
+| [Client](/javascript-api/k6-experimental/grpc/client) | gRPC client used for making RPC calls to a gRPC Server. |
+| [Client.load(importPaths, ...protoFiles)](/javascript-api/k6-experimental/grpc/client/client-load) | Loads and parses the given protocol buffer definitions to be made available for RPC requests. |
+| [Client.connect(address [,params])](/javascript-api/k6-experimental/grpc/client/client-connect) | Connects to a given gRPC service. |
+| [Client.invoke(url, request [,params])](/javascript-api/k6-experimental/grpc/client/client-invoke) | Makes a unary RPC for the given service/method and returns a [Response](/javascript-api/k6-experimental/grpc/response). |
+| [Client.close()](/javascript-api/k6-experimental/grpc/client/client-close) | Close the connection to the gRPC service. |
+| [Params](/javascript-api/k6-experimental/grpc/params) | RPC Request specific options. |
+| [Response](/javascript-api/k6-experimental/grpc/response) | Returned by RPC requests. |
+| [Constants](/javascript-api/k6-experimental/grpc/constants) | Define constants to distinguish between [gRPC Response](/javascript-api/k6-experimental/grpc/response) statuses. |
+| [Stream](/javascript-api/k6-experimental/grpc/stream) | Creates a new GRPC stream. |
+| [Stream.on](/javascript-api/k6-experimental/grpc/stream/stream-on) | Adds a new listener to one of the possible stream event's. |
+| [Stream.write](/javascript-api/k6-experimental/grpc/stream/stream-write) | Writes a message to the stream. |
+| [Stream.end](/javascript-api/k6-experimental/grpc/stream/stream-end) | Signals to server that client finished sending. |
```

Comment:
```suggestion
| [Stream.on](/javascript-api/k6-experimental/grpc/stream/stream-on) | Adds a new listener to one of the possible stream event's. |
| [Stream.write()](/javascript-api/k6-experimental/grpc/stream/stream-write) | Writes a message to the stream. |
| [Stream.end()](/javascript-api/k6-experimental/grpc/stream/stream-end) | Signals to server that client finished sending. |
```
This sounds more consistent with the Close method and with the title in the page itself.